handler.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ccxt = require('ccxt');
+const axios = require('axios');
+
+module.exports.transaction = async (event) => {
+  var jobId = event.Records[0].Sns.Message;
+
+  var jobURL = `http://jobs.service.thinblock.io/jobs/${jobId}?look_up=_id`;
+  const { data: { actions: [ action ] } } = await axios.get(jobURL);
+
+  const { params } = action;
+
+  if (params.user_id === undefined || params.exchange === undefined ||
+    params.symbol === undefined || params.side === undefined ||
+    params.amount === undefined || params.price === undefined) {
+    return {
+      statusCode: 400,
+      body: JSON.stringify({
+        message: 'Requires user_id, exchange, symbol, side, amount, price'
+      })
+    };
+  }
+
+  const user = params.user_id;
+  const exchangeName = params.exchange;
+  let { symbol, side, amount, price } = params;
+  let order = null;
+
+  try {
+    // add URL
+    var url = `http://pair.service.thinblock.io/api/pairs?user_id=${user}&exchange=${exchangeName}`;
```

Comment:
Can we get this from `env` variable? Otherwise its gonna be messy changing the code. We can do a check if it exists in env variable otherwise this url.
`
```suggestion
    const base = process.env.KEY_SERVICE_BASE_URL || 'http://pair.service.thinblock.io'
    const url = `${base}api/pairs?user_id=${user}&exchange=${exchangeName}`
```